### ISSUE TITLE
Incompatibility with Additional Enchanted Miner 1.18 v18.3.549

### ISSUE THREAD

**Issue by Loyal0713** [Globox1997/LevelZ#126]
Not sure if I'm misunderstanding how this works. I'm currently level 4 which means I should be able to mine stone, however using either wood or stone pickaxes mines slow and they don't drop cobblestone. Also I'm not seeing cobblestone on the mining list (I assume level 3).

Current stats:
![image](https://user-images.githubusercontent.com/42594335/165135516-95878094-1361-4143-8c1e-6480d50d80fd.png)

Mining list:
![image](https://user-images.githubusercontent.com/42594335/165135625-f2633b7a-77dc-47ce-a9e8-9397f9f41c25.png)

**Comment by Loyal0713** [comment]
Demo video
https://youtu.be/CXJyyWjPWtE

**Comment by Globox1997** [comment]
Does it happen when only levelz is installed?

**Comment by Loyal0713** [comment]
Running using Valhelsia Enhanced Vanilla 1.2.0-pre3 with some additional mods. Just checked running with the default Valhelsia modpack (no additional) and it appears to work just fine. Below is a list of the mods I added to Valhelsia. Will try to go through and see which one(s) causes the issue starting with more obvious mods although a quick glance nothing appears to be incompatible

	BasicShields-1.3.1-1.18.2
	Clumps-fabric-1.18.2-8.0.0+8
	CosmeticArmor-1.18.2-1.3.0
	Croptopia-1.18.2-FABRIC-1.8.3
	DarkUtilities-Fabric-1.18.2-10.0.4
	FabricShieldLib-1.5.3-1.18
	NaturesCompass-1.18.2-2.0.3-fabric
	Oh_The_Biomes_You'll_Go-fabric-1.18.2-1.4-RC-10
	PigPen-Fabric-1.18.2-8.0.1
	Runelic-Fabric-1.18.2-11.0.1
	TerraBlender-fabric-1.18.2-1.1.0.99
	The_Graveyard_1.7_(FABRIC)_for_1.18.2
	badpackets-fabric-0.1.1
	bclib-1.4.3
	better-nether-6.1.0
	betteranimalsplus-1.18.2-11.0.5-fabric
	caracal-1.18.2-1.3.8
	dark-enchanting-0.7.0+1.18.2
	dimension-coord-converter-1.1.1
	display-death-coordinate-1.0.0
	elytratrinket-fabric-3.0.0-1.18.1
	extraalchemy-fabric-1.18.2-1.8.1
	feature_nbt_deadlock_be_gone_fabric-2.0.0+1.18.2-rc1
	friendsandfoes-fabric-mc1.18.2-1.3.4
	grizzlybear-1.18-1.3
	impaled-1.1.1
	inmis-2.5.3-1.18.2
	lambdynamiclights-2.1.0+1.17
	leaves-us-in-peace-1.0.4+MC1.18-1.18.2
	light-overlay-6.0.5
	malilib-fabric-1.18.2-0.12.1
	mavm-fabric-1.18.1-1.0.7a
	medievalweapons-1.1.10
	morevillagers-FABRIC-1.18.2-3.2.0
	mostructures-1.4.1+1.18.2
	mythic-mounts-1.18.2-6.0
	reeses_sodium_options-mc1.18.2-1.4.2
	repurposed_structures_fabric-4.1.6+1.18.2
	sodium-extra-mc1.18.2-0.4.3
	stoneholm-1.4.3
	things-0.2.17+1.18
	totw_additions-1.2.1-1.18.2-fabric-polymer
	tweakeroo-fabric-1.18.2-0.13.1
	villagernames-1.18.2-4.0.4
	voidtotem-fabric-1.18.2-1.3.1
	voidz-1.0.6
	wraith-waystones-2.5.0

**Comment by Loyal0713** [comment]
Progress, reinstalled all the mods while occasionally testing if breaking blocks works. All the mods I personally downloaded still resulted in blocks dropping/breaking normally. I then went through the updates one by one, and turns out it was an update for Additional Enchanted Miner (AdditionalEnchantedMiner-1.18-fabric-18.3.549.jar). Confirmed by updating everything including AEM, disabling it, then successfully mining.
